### PR TITLE
Relocated blacklist

### PR DESCRIPTION
**Pull request by lkalinka**

### CODE REVIEW

File: EventListener/DiscoverUrlListener.php [hunk 1]
```diff
@@ -22,15 +22,45 @@ class DiscoverUrlListener
...
+
+    /**
+     * Check if given url is blacklisted
+     *
+     * @param string $url
+     * @param array $blacklist
+     *
+     * @return boolean
+     */
+    public function isUrlBlacklisted($url, $blacklist)
```
Comment (keesschepers) [inline EventListener/DiscoverUrlListener.php]: Can you typehint the $blacklist variable?

File: composer.json [hunk 2]
```diff
@@ -9,6 +9,10 @@
         {
             "name": "Nick van Kaam",
             "email": "n.vankaam@simgroep.nl"
+        },
+        {
+            "name": "Lukasz Kalinka",
+            "email": "l.andrzej@simgroep.nl"
```
Comment (keesschepers) [inline composer.json:15]: :sunglasses:

File: EventListener/DiscoverUrlListener.php [hunk 3]
```diff
@@ -49,6 +79,17 @@ public function onDiscoverUrl(Event $event)
                 $uri = new Uri(substr($uri, 0, $position));
             }
 
+            $isBlacklisted = $this->isUrlBlacklisted($uri->normalize()->toString(), $crawlJob->getBlacklist());
+
+            if ($isBlacklisted) {
+                $this->eventDispatcher->dispatch(
+                    "spider.crawl.blacklisted",
+                    new Event($this, array('uri' => $uri))
```
Comment (nickvkaam) [inline EventListener/DiscoverUrlListener.php]: old array notation

Comment (nickvkaam) [inline EventListener/DiscoverUrlListener.php]: Old array notation

**Pull request merged**